README.md

```diff
@@ -75,39 +75,41 @@ Examples:
 ```
 ### Using a custom config:
 You can use a custom json file named ```ccube_config.json``` to store configs for ccube,i.e:
-```
+```json
 {
-
- "noLoopSwitch":"true",
+ "noLoopSwitch":true,
  "number":"69",
  "filePath":"void"
-
 }
 ```
 you can change the values to your need.
 
-**Note:Don't change any of the keys(noLoopSwitch,etc) or your config won't work** 
+**Note: Don't remove any of the keys (noLoopSwitch or others) or your config won't work** 
 
-### Build from source:
+### Building from source
 **Build requirements:**
-- Nim(advised to use latest version)
+- Nim (use the latest version for the best results)
 
-To build it from scource.simply: 
+To build it from source, simply: 
 - Clone this repository and cd to the folder.
-- Build it using ```nim c ccube.nim```.if you want to gain optimal speed then use ```-d:danger``` switch which disables all the runtime checks,for more [info](https://nim-lang.org/faq.html).
+- Run ```nim c ccube.nim```. If you want to have the best performance then use `-d:danger` switch which disables all the runtime checks, for more info see [the Nim FAQ](https://nim-lang.org/faq.html). 
+
+### Additional performance flags 
+If your Nim version is 1.4.0 or higher, you can compile using `-d:danger --gc:orc` which is even faster.
+Also, if you don't mind longer compile times, you can use link-time optimization: `nim c -d:danger --gc:orc -d:lto ccube.nim`.
```

Comment:
See https://en.wikipedia.org/wiki/Interprocedural_optimization#WPO_and_LTO for more, basically Nim just uses the C compiler's lto flags for compilation in this case which results in faster binaries.

Reply:
yeah i saw that,only used those switches for windows binary